x/auth/ante/testutil_test.go

```diff
@@ -0,0 +1,116 @@
+package ante_test
+
+import (
+	"errors"
+	"fmt"
+
+	"github.com/stretchr/testify/suite"
+	"github.com/tendermint/tendermint/crypto"
+
+	"github.com/cosmos/cosmos-sdk/client"
+	"github.com/cosmos/cosmos-sdk/client/tx"
+	"github.com/cosmos/cosmos-sdk/simapp"
+	simappparams "github.com/cosmos/cosmos-sdk/simapp/params"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/cosmos/cosmos-sdk/types/tx/signing"
+	"github.com/cosmos/cosmos-sdk/x/auth/ante"
+	xauthsigning "github.com/cosmos/cosmos-sdk/x/auth/signing"
+	"github.com/cosmos/cosmos-sdk/x/auth/types"
+)
+
+// TestAccount represents an account used in the tests in x/auth/ante.
+type TestAccount struct {
+	acc  types.AccountI
+	priv crypto.PrivKey
+}
+
+// AnteTestSuite is a test suite to be used with ante handler tests.
+type AnteTestSuite struct {
```

Comment:
I'm not 100% sure where the idiomatic place to put this. It's a suite used by `*_test.go` in the same folder.